news/2023/2023-11-08-OWC2023-RO16-recap.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "osu! World Cup 2023: Round of 16 Recap"
+date: 2023-11-08 16:30:00 +0000
+---
+
+Want to stay up to date with all the latest osu! World Cup happenings, or maybe just look back at the action packed weekend? [Junihuhn](https://osu.ppy.sh/users/4182339) is here to deliver!
+
+![](/wiki/Tournaments/OWC/2023/img/owc2023-banner.jpg)
+
+As previously mentioned in [this news post](https://osu.ppy.sh/home/news/2023-08-21-contributing-to-the-osu-news-feed), this year you can expect a weekly rundown of the osu! World Cup stages, starting with the Round of 16. Hosted by the one and only Junihuhn, they will be taking the best moments of the previous week's stage and *burning them into the retina's of your eyes*.
+
+Also, as is tradition by now they will also be doing a video series on their [YouTube channel](https://www.youtube.com/@Junihuhn/) (which is a mandatory viewing for all osu! World Cup enthusiasts). You can find the [Round of 32 recap](https://www.youtube.com/watch?v=ydOlxHQg0UM) from the previous week on there, or you can check out the Round of 16 recap in a video format below:
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/SlWKKA-ltZY" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Without further ado, let's look at some highlighted matches from the Round of 16!
+
```

Comment:
bit redundant and also awkwardly sandwiched between the video above and the heading below

![image](https://github.com/ppy/osu-wiki/assets/36758269/c055455b-2f3a-4bb5-aabd-068e8e9f089b)

```suggestion
```